ReleaseChecklist.md

```diff
@@ -20,50 +20,52 @@
 
 ### Create the Release
  - [ ] Create Github release page: https://github.com/ethereum/solidity/releases/new
- - [ ] On the release page, select the ``release`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
- - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e origin/release..origin/develop``).
- - [ ] Create a pull request from ``develop`` to ``release``, wait for the tests, then merge it.
+ - [ ] On the release page, select the ``develop`` branch as new target and set tag to the new version (e.g. `v0.5.4`) (make sure you only `SAVE DRAFT` instead of `PUBLISH RELEASE` before the actual release)
+ - [ ] Thank voluntary contributors in the Github release page (use ``git shortlog -s -n -e v0.5.4..origin/develop``).
  - [ ] Make a final check that there are no platform-dependency issues in the ``solidity-test-bytecode`` repository.
- - [ ] Wait for the tests for the commit on ``release``, create a release in Github, creating the tag (click the `PUBLISH RELEASE` button on the release page.)
- - [ ] Wait for the CI runs on the tag itself (travis should push artifacts onto the Github release page).
+ - [ ] Check that all tests on the latest commit in ``develop`` are green.
+ - [ ] Click the `PUBLISH RELEASE` button on the release page, creating the tag.
+ - [ ] Wait for the CI runs on the tag itself (travis will push the source archive and the static linux binary onto the Github release page).
+
+### Download Binaries
  - [ ] Take the ``solc.exe`` binary from the ``b_win_release`` run of the released commit in circle-ci and add it to the release page as ``solc-windows.exe``.
- - [ ] Run ``scripts/create_source_tarball.sh`` while being on the tag to create the source tarball. Make sure to create ``prerelease.txt`` before: (``echo -n > prerelease.txt``). This will create the tarball in a directory called ``upload``.
- - [ ] Take the tarball from the upload directory (its name should be ``solidity_x.x.x.tar.gz``, otherwise ``prerelease.txt`` was missing in the step before) and upload the source tarball to the release page.
+ - [ ] Take the ``solc`` binary from the ``b_osx`` run of the released commit in circle-ci and add it to the release page as ``solc-macos``.
+ - [ ] If not done by travis: Take the ``soljson.js`` binary from the ``b_ems`` run of the released commit in circle-ci and add it to the release page as ``soljson.js``.
+
+### Update solc-bin
```

Comment:
Not really necessary as you're the one making releases but in case anyone else has to do it a link to solc-bin would make it easier to find out that it's a separate repo if one does not know that yet :)

```suggestion
### Update [solc-bin](https://github.com/ethereum/solc-bin/)
```